README.md

```diff
@@ -187,6 +187,12 @@ ensure consistency and readability in jest test suites.
 
 https://github.com/dangreenisrael/eslint-plugin-jest-formatting
 
+### eslint-plugin-istanbul
+
+A set of rules to enforce good practices for istanbuljs.
```

Comment:
```suggestion
A set of rules to enforce good practices for Istanbul.
```

We should say something about how it's for code coverage, perhaps

Reply:
What do you think about this?
```
A set of rules to enforce good practices for Istanbul, a tool used to measure test coverage within Jest.
```

Reply:
I like it! Maybe "the tool" instead of "a tool"?

Reply:
that's not true for v8 code coverage, though... Maybe "the tool used ... expect when using V8 code coverage" or something like that?

@coreyfarrell ideas?

Reply:
"... Istanbul, the default coverage collection tool"?  Sorry not sure I have good ideas.

Reply:
we'd keep using istanbul for reporting and stuff even if we stop using it for instrumenting. Maybe if we say "... Istanbul, the coverage reporting tool" or something? Not sure if it really matters

Reply:
These hints are not processed by the istanbul reporting modules, they are only processed during instrumentation.  This is contrary to c8 which *I think* processes `c8 ignore next` during the v8-to-istanbul step.

Reply:
How about:

```
A set of rules to enforce good practices for Istanbul, one of the code coverage tools used by Jest.
```

Also, is this is the only thing blocking? Because I think we might be bikeshedding a bit :joy: